workshops/personal_website/README.md

```diff
@@ -0,0 +1,239 @@
+# Personal Website
+
+In this workshop, you will learn how to make a website from scratch.
+
+Here is the final product you will end up with:
+
+![](http://placehold.it/200?text=Preview)
+
+> Open the [live demo](#TODO).
+>
+> View [the code](#TODO).
+
+## Part I: Setup Your Tools
+
+### GitHub
+
+- What is GitHub?
+- What are we using GitHub for?
+- Let's setup your GitHub account.
+- Write down your email, username, and password somewhere you won't forget
+  (you'll need it later in this tutorial and later in the semester).
```

Comment:
I don't think it's necessary to have them write down their login info. I think we can trust people to use stuff they'll remember.